tony-history-server/app/controllers/JobConfigPageController.java

```diff
@@ -32,16 +35,21 @@ public JobConfigPageController(Config config) {
   public Result index(String jobId) {
     HdfsConfiguration conf = Configuration.getHdfsConf();
     FileSystem myFs = HdfsUtils.getFileSystem(conf);
+    Cache<String, List<JobConfig>> cache = CacheWrapper.getConfigCache();
 
     if (myFs == null) {
       return internalServerError("Failed to initialize file system");
     }
 
-    List<JobConfig> listOfConfigs = new ArrayList<>();
+    List<JobConfig> listOfConfigs;
     String tonyHistoryFolder = config.getString("tony.historyFolder");
     Path xmlPath = new Path(tonyHistoryFolder + jobId + "/config.xml");
```

Comment:
I think I mentioned this elsewhere, but safer to add "/" between `tonyHistoryFolder` and `jobId` in case `tonyHistoryFolder` doesn't have a trailing slash.

Reply:
Yeah I think so too. I think it was before the new metadata parser commits, where I metadata was written to a file instead of encoded in the file name. Anyhow, fixed: https://github.com/linkedin/TonY/pull/85/commits/9f2c11c2422590396cf81c48e241ae952ccce85e